backend/webapi/Features/EndorsementRequests/Create.cs

```diff
@@ -75,10 +75,29 @@ public async Task<Model> HandleAsync(Command command)
                 return new(true);
             }
 
-            var partyName = await this.context.Parties
+            var parties = await this.context.Parties
```

Comment:
As the Where condition to filter by recipient email is added down(in line 96) we can remove this Where condition.